### ISSUE TITLE
[A11y_Bug]: Putting the website content in accordions makes it difficult to search for items

### ISSUE THREAD

**Issue by stefaniavaloroso** [IBMa/equal-access#1013]
### Project

able site

### Browser

Chrome

### Operating System

MacOS

### Description

I'm trying to find content on your accessibility site and because the content is hidden within accordions, I cannot use the search feature in my browser to look for what I need. this causes me to expand every single section in order to search it, which is pretty ironic.

At a minimum please provide an "expand all sections" option at the top of the page.

Additionally, using the website search bar is confusing - if something matches your term, it automatically highlights the first result in the list. So when you go to press enter to complete your search term, you're brought to that page instead of a "results" page to browse through.

### Steps to reproduce

1. open any section (I used https://www.ibm.com/able/toolkit/design/ux/keyboard/)
2. press ctrl + f in your browser to open the search dialogue
3. search for a term that isn't in the accordion headers (this is condensed information and doesn't always have the terms someone is looking for)

**Comment by aliunwala** [comment]
Triage:
This issue has been reopened in the relevant repo internally with the team that will work on making this happen:
https://github.ibm.com/IBMa/E2E/issues/4756

This is an internal link that is not accessible by non-IBM employees. I will cross link that issue to this one as well.

**Comment by MHoov** [comment]
@aliunwala internal issue 4756 is closed, please update this issue accordingly

**Comment by aliunwala** [comment]
This has been updated and propagated to the main site in various locations. 

Here is an example page with the expand all for the accordions on it:
https://www.ibm.com/able/toolkit/design/content/
